pkgs/servers/mastodon/default.nix

```diff
@@ -84,6 +84,7 @@ stdenv.mkDerivation rec {
     fi
     ln -s ${mastodon-assets}/public/assets public/assets
     ln -s ${mastodon-assets}/public/packs public/packs
+    ln -s /var/lib/mastodon/public-system public/system
```

Comment:
We shouldn't link the outside world into the Nix store. This creates impurity.

Reply:
Then how do you fix the error?